package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "js-miniapp",
+  "version": "0.1.0",
+  "description": "Mini App SDK for JavaScript",
+  "main": "build/index.js",
```

Comment:
I ran `npm run compile` to generate the compiled JS files and the index file was located at `build/src/index.js`. I don't know which we'd want to shoot for, but I thought you should know.

Reply:
Oh, you're right. I think this is an issue with `tsconfig`. It needs to use `./src` as the root.

Also, isn't it conventional to use `dist` as the distribution build folder? Maybe we should use that instead of `build`?

Reply:
Yup, setting `src` as the root directory makes sense. In that case, you probably have to move the test files in there too since they're written in TypeScript (I think there will be issues it they live outside the root directory).

I'm not sure what the convention is, but I've seen `dist` more than `build` lately. I think either will work as long all the pieces are follow one or the other.

Reply:
guess, `build` comes baked with `gts`(so google of them lol)